EIPS/eip-5656.md

```diff
@@ -0,0 +1,104 @@
+---
+eip: 5656
+title: Memory copying instruction
```

Comment:
```suggestion
title: Optcode Instruction for Memory Copy
```

Reply:
I think this is unnecessary + it's "opcode" not "opt".

Reply:
Makes sense @lightclient.
I withdraw my suggested revision.